sfm/sfm/settings/common.py

```diff
@@ -193,6 +193,9 @@
 EMAIL_HOST_PASSWORD = env.get('SFM_EMAIL_PASSWORD')
 EMAIL_USE_TLS = True
 
+# contact email
+CONTACT_EMAIL = env.get('SFM_CONTACT_EMAIL', 'sfm@gwu.edu')
```

Comment:
Should not default to sfm@gwu.edu.  Can either make required or optional.  If optional and not provided, then display is skipped.